soc/nordic/Kconfig

```diff
@@ -73,6 +73,13 @@ config NRF_ACL_FLASH_REGION_SIZE
 	help
 	  FLASH region size for the NRF_ACL peripheral.
 
+config NRF_RRAM_REGION_SIZE_GRANULARITY
```

Comment:
I think a better place for this value definition is in `drivers/flash/Kconfig.nrf_rram`